sketch.js

```diff
@@ -10,9 +10,18 @@ var decaySlider, decay;
 var sustainSlider, sustain;
 var releaseSlider, release;
 
+var keyWidth, keyHeight;
+var xTranslate, yTranslate;
```

Comment:
Can you rename these something more descriptive? Since these translates are only used for the keyboard elements it is somewhat confusing. Perhaps `xTranslateKeys` and `yTranslateKeys`